raiden/tests/smart_contracts/test_channel_manager.py

```diff
@@ -134,14 +139,71 @@ def test_channelmanager(tester_state, tester_token, tester_events,
         'settleTimeout': settle_timeout,
     }
 
-    # uncomment private in function to run test
-    # assert channel_manager.numberOfItems(netting_channel_creator1) == 2
-    # assert channel_manager.numberOfItems(sha3('address1')[:20]) == 1
-    # assert channel_manager.numberOfItems(sha3('iDontExist')[:20]) == 0
-    # vs = sorted((sha3('address1')[:20], sha3('address2')[:20]))
-    # k0 = channel_manager.key(sha3('address1')[:20], sha3('address2')[:20])
-    # assert k0 == sha3(vs[0] + vs[1])
-    # k1 = channel_manager.key(sha3('address2')[:20], sha3('address1')[:20])
-    # assert k1 == sha3(vs[0] + vs[1])
-    # with pytest.raises(TransactionFailed):
-    #    channel_manager.key(sha3('address1')[:20], sha3('address1')[:20])
+
+def test_deleteChannel(tester_state, tester_channelmanager, tester_channels, settle_timeout,
```

Comment:
style: if it's not too long, just put all arguments in one line. If it is quite long, split each argument on its own line.